addons/oceanbase-ce/dataprotection/backup.sh

```diff
@@ -1,135 +1,45 @@
-set -e
+# !/bin/bash
```

Comment:
`# !/bin/bash` ->  `#!/bin/bash`  remove the blank

Reply:
fixed